sova/dashboard/services/fleet_service.py

```
@@ -0,0 +1,450 @@
+            return self._merge([], skipped, [], [], [], [])
+
+        sem = asyncio.Semaphore(_MAX_CONCURRENT_SCANS)
+
+        async def _safe_query(slug: str, db_path: Path) -> tuple[str, _ProjectResult | None]:
+            async with sem:
+                try:
+                    result = await asyncio.wait_for(
+                        self._query_project(slug, db_path),
+                        timeout=self._cfg.query_timeout_seconds,
+                    )
+                    return slug, result
```

Ruff (0.15.21)

[warning] Consider moving this statement to an `else` block

(TRY300)